scripts/view.js

```diff
@@ -824,7 +826,30 @@ view.settings = {
 			settings.bind('#JustifiedLayout','.setLayoutOverlay',settings.changeLayout);
 			settings.bind('#ImageOverlay','.setLayoutOverlay',settings.changeImageOverlay);
 		},
-		
+
+		setOverlayType: function() {
+			let msg =`
+			<div class="setOverlayType">
+			<p>${ lychee.locale['LAYOUT_TYPE'] }
+			<span class="select" style="width:270px">
```

Comment:
Rather than a select, a toggle would be better here:

- if exif is toggled:
```
exif (o---) desc
```

- if desc is toggled:
```
exif (---o) desc
```
It is more visual than a drop down list.

Have a look at how the setting for enabling the overlay is done. :)
Also making a dependency on those like if the overlay setting is on,
the switch (exif - desc) is "available", if the overlay setting is off, the switch is grayed.

Reply:
After further thought. Let's keep it with a `<select><options>...` But let's also add as a possible overlay the following one (if the taken date is available: `cameraDate == '1'`):
```
Title
taken date - time
```

Reply:
We need to update the `Photo` PHP model to display the time. Currently, it returns a datestring rather than a date object. The easier method would be to return the date object and manipulate the date in Lychee Front using JS, but I don't know how that affects long-term changes. Thoughts?

Reply:
I am not sure I get what you mean?
value `takedate` is properly formatted e.g. "30 November 2018 at 03:34"

Reply:
Weird...mine don't show a time. Doesn't the `Photo` only return day - month - year?

https://github.com/LycheeOrg/Lychee/blob/master/php/Modules/Photo.php#L889

Reply:
okay
```php
$photo['takedate']    = strftime('%d %B %Y', $data['takestamp']);
```
that's a mistake, we should also add the time in it. :+1:

Reply:
I'll add it to the other PR already open for this update.

Reply:
12 hour time (for us Americans 😉) or 24 hour?

Reply:
24 hours, we are civilized here. :D